lang/python/python-aiohttp-cors/Makefile

```diff
@@ -0,0 +1,45 @@
+#
+# Copyright (C) 2019 CZ.NIC, z. s. p. o. (https://www.nic.cz/)
+#
+# This is free software, licensed under the GNU General Public License v2.
+# See /LICENSE for more information.
+#
+
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=aiohttp-cors
+PKG_VERSION:=0.7.0
+PKG_RELEASE:=1
+
+PKG_SOURCE:=aiohttp-cors-$(PKG_VERSION).tar.gz
+PKG_SOURCE_URL:=https://files.pythonhosted.org/packages/source/a/aiohttp_cors/
+PKG_HASH:=4d39c6d7100fd9764ed1caf8cebf0eb01bf5e3f24e2e073fda6234bc48b19f5d
+
+PKG_MAINTAINER:=Josef Schlehofer <pepe.schlehofer@gmail.com>
+PKG_LICENSE:=Apache-2.0
+PKG_LICENSE_FILES:=LICENSE
+
+include $(INCLUDE_DIR)/package.mk
+include ../python3-package.mk
+
+define Package/python3-aiohttp-cors
+  SECTION:=lang
+  CATEGORY:=Languages
+  SUBMENU:=Python
+  TITLE:=CORS support for aiohttp
+  URL:=https://github.com/aio-libs/aiohttp-cors
+  DEPENDS:= \
+	+python3-light \
+	+python3-setuptools \
+	+python3-asyncio \
```

Comment:
It looks like [asyncio is only directly used in some test cases](https://github.com/aio-libs/aiohttp-cors/search?q=asyncio)?

Reply:
Also to clarify, if this is only used for test cases then it isn't necessary as an install dependency either.

Reply:
```
root@turris:~# opkg list-installed | grep python3
python3-aiohttp - 3.5.4-1
python3-aiohttp-cors - 0.7.0-1
python3-attrs - 19.1.0-3.7-1.3
python3-base - 3.7.2-3.7-13.3
python3-codecs - 3.7.2-3.7-13.3
python3-email - 3.7.2-3.7-13.3
python3-idna - 2.8-3.7-2.3
python3-light - 3.7.2-3.7-13.3
python3-multidict - 4.5.2-3.7-1.3
python3-urllib - 3.7.2-3.7-13.3
python3-yarl - 1.3.0-3.7-1.3
```

```
>>> import aiohttp_cors
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/__init__.py", line 20, in <module>
  File "/__init__.py", line 6, in <module>
  File "/client.py", line 32, in <module>
  File "/http.py", line 7, in <module>
  File "/http_parser.py", line 14, in <module>
  File "/base_protocol.py", line 4, in <module>
  File "/tcp_helpers.py", line 20, in <module>
AttributeError: module 'asyncio' has no attribute 'Transport'
```

However, asyncio is dependency in aiohttp.

Reply:
I think my [other comment](https://github.com/openwrt/packages/pull/8734#discussion_r300765548) applies here as well.

Reply:
Asyncio removed. Now, there are just two dependencies `python3-light` and `python3-aiohttp`.